content/articles/introduction-to-php-graphic-techniques/index.md

```diff
@@ -0,0 +1,223 @@
+ Introduction To PHP Graphic Techniques
+### Introduction
+PHP has the inherent ability to create and manipulate graphics independently. The scope and capabilities of PHP graphic handling is quite deep and powerful. It does this with the aid of the GD library which currently supports image formats in jpeg, png, gif and wbmp. 
+This tutorial aims at providing a solid foundation on the first steps to take to start doing exploits with PHP graphics. If you’re as excited as I am, then let’s proceed. 
+
+### Prerequisites
+To follow along with this tutorial, you need to have the following:
+•	 A code editor like Visual Studio Code[link].
+•	 Apache web server, Preferably WAMP[link] or XAMPP[link].

```

Comment:
XAMPP/WAMPP is a stack, are they really servers?

Reply:
I'm sorry for the mistake, it was a typographical error. I've corrected it in the updated file.